sql/migration_version_cleanup.sql

```diff
@@ -0,0 +1,6 @@
+-- The item.version 45 does not have a migration version.  This only happened due to development processes, and we need
```

Comment:
Developers can run this locally and in dev/QA.

**This does not need to be run in staging or production**